.buildkite/pipeline.yml

```diff
@@ -216,6 +216,18 @@ steps:
         artifact_paths: "sphere_baroclinic_wave_rhoe_equilmoist/*"
         agents:
           slurm_constraint: icelake|cascadelake|skylake|epyc
+      
+      - label: ":computer: no lim ARS baroclinic wave (ρe) equilmoist (deep sphere)"
+        command: >
+          julia --color=yes --project=examples examples/hybrid/driver.jl
+          --config_file $CONFIG_PATH/deep_sphere_baroclinic_wave_rhoe_equilmoist.yml
+
+          julia --color=yes --project=examples regression_tests/test_mse.jl
+          --job_id deep_sphere_baroclinic_wave_rhoe_equilmoist
+          --out_dir deep_sphere_baroclinic_wave_rhoe_equilmoist
+        artifact_paths: "deep_sphere_baroclinic_wave_rhoe_equilmoist/*"
+        agents:
+          slurm_constraint: icelake|cascadelake|skylake|epyc
```

Comment:
I added this test to ensure we are using the deep sphere code, but if it doesn't make sense physically, we should remove it and put something else in place.